forem/forem
# Move footer to sidebar on home/post index page

## What type of PR is this? (check all applicable)


## Description

## Related Tickets & Documents



- Closes #19583

## QA Instructions, Screenshots, Recordings

[Loom Recording](https://www.loom.com/share/b6d87a6d09164382b6363199a04cb346?sid=260bccca-6a22-41fc-84c4-68bbbd2e143b)

1. Visit home page, try scrolling infinitely and notice that the footer is not visible at bottom.
2. The footer should be visible on `left-sidebar` on home page below all the billboards.
3. On any other child page, like article, podcast, video, etc the footer should be visible at bottom only.

<img width="1512" alt="Screenshot 2023-07-20 at 1 59 58 PM" src="https://github.com/forem/forem/assets/9396084/a9626261-886d-4859-af9f-5d9b8a5a531c">


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

      have not been included_

## [optional] Are there any post deployment tasks we need to perform?

There might be some cashing issue for users.
